### DISCUSSION TITLE
When I revalidated my academic status, I ran into problems.

### DISCUSSION THREAD

**Discussion by Wenrh2004** [community/community#67122] [GitHub Education]
### Hi everyone,

当我在GitHub教育页面上时，它提示我需要重新验证我的学术状态，我选择在页面上提供的链接进行验证，但在相关验证方面，他告诉我联系支持，但我试图联系支持，但没有得到回复。
<img width="1304" alt="截屏2023-09-15 01 45 38" src="https://github.com/community/community/assets/122615371/d29c4d79-7603-4511-bee4-c47858953554">

**Comment by Wenrh2004** [comment]
I am a verified student having trouble redeeming my Global Campus offers

Description

There are outstanding issues with your GitHub account. Please contact GitHub Support for more information.

1696918097@qq.com has already been used to obtain an academic discount on GitHub account(s) Wenrh2004.

<img width="1304" alt="截屏2023-09-15 00 51 21" src="https://github.com/community/community/assets/122615371/ee68eafa-3bd3-4185-9914-cd503b43211f">

**Comment by chenjiefeng2001** [comment]
你可以参考一下这个discussion下的回答，希望对你有帮助
https://github.com/orgs/community/discussions/49494

**Reply by Wenrh2004** [reply]
我尝试联系支持了，但是我的单号一直没有回复，导致我并不知道我的账号问题出现在哪里，而且它页面的问题完全是冲突的，所以我也很不理解，而且我也尽可能的表达了我的诉求，但是我的工单一直没有得到回复 @samus-aran 
这是我提交的工单，相关问题都已经截屏了，我也很遗憾，我每天花费大量精力用于处理GitHub学术认证的问题，并且我很难得到官方的帮助
<img width="1304" alt="截屏2023-09-15 02 18 04" src="https://github.com/community/community/assets/122615371/a5b1b327-89eb-4405-a270-708ef8dfe2a9">
